scripts/test_start.sh

```diff
@@ -10,33 +10,36 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I am lost....

Reply:
The important change in test_start.sh is that sometimes faucet started too early, at a point where the blockchain was not ready. I moved the faucet starts down to givve the blockchains some time to get warm. A few seconds are enough.

Reply:
Perfect.